lib/src/test/java/vmath/Mat3Test.java

```diff
@@ -153,4 +153,18 @@ void whenToArrayCalled_thenConvertsToArray() {
         assertArrayEquals(a1, a2);
         assertNotSame(a1, a2);
     }
+
+    @Test
+    void whenToStringCalled_thenReturnsStringRepresentation() {
+    var m = new Mat3(
+        1.0f, 2.0f, 3.0f,
+        4.0f, 5.0f, 6.0f,
+        7.0f, 8.0f, 9.0f
+    );
+    String expected = "(1.0, 2.0, 3.0)\n" +
+                      "(4.0, 5.0, 6.0)\n" +
+                      "(7.0, 8.0, 9.0)";
+    
+    assertEquals(expected, m.toString());
+    }
```

Comment:
Look how the Mat2 `toString` tests were [done](https://github.com/toksaitov/vmath/blob/db08f086a9b55948a9c395c9913c155e417d1e01/lib/src/test/java/vmath/Mat2Test.java#L153-L177) and do the same.